### ISSUE TITLE
Add config option for ZeroUnit to exempt certain properties

### ISSUE THREAD

**Issue by cvrebert** [brigade/scss-lint#702]
There are some cases where it's necessary to include a unit for `0` values of certain properties, for the sake of compatibility with buggy browsers (e.g. `flex` for IE10-11 per https://github.com/philipwalton/flexbugs/blob/master/README.md#4-flex-shorthand-declarations-with-unitless-flex-basis-values-are-ignored ). It would therefore be nice if the `ZeroUnit` linter had a config option which was a list of properties to exempt from the unitless-zeros requirement.

**Comment by thejamespower** [comment]
Agreed, I believe zero is a valid non-variable value and the option to ignore it would be extremely useful.
